Doc/library/csv.rst

```diff
@@ -408,9 +411,12 @@ Dialects support the following attributes:
 
 .. attribute:: Dialect.escapechar
 
-   A one-character string used by the writer to escape the *delimiter* if *quoting*
-   is set to :const:`QUOTE_NONE` and the *quotechar* if *doublequote* is
-   :const:`False`. On reading, the *escapechar* removes any special meaning from
+   A one-character string used by the writer to escape the *delimiter*,
+   the *quotechar*, ``'\r'``, ``'\n'`` and any of the characters in
+   *lineterminator* if *quoting* is set to :const:`QUOTE_NONE`,
+   the *quotechar* if *doublequote* is :const:`False`,
```

Comment:
I do not understand this part of the sentence in the original, nor the addition below.

Reply:
How would you write it?

Different characters are escaped depending on different conditions:

* if *quoting* is set to :const:`QUOTE_NONE`, it escapes *delimiter*, *quotechar*, etc.
* if *doublequote* is False, it escapes *quotechar*.

Reply:
Perhaps:

```
A one-character string for escaping. If quoting is set to :const:QUOTE_NONE, it escapes delimiter, quotechar,  ``'\r'``, ``'\n'`` and any of the characters in *lineterminator*. If doublequote is False, it escapes quotechar
```